jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditorDialog.kt

```diff
@@ -49,8 +51,12 @@ class QueryEditorDialog(
     }
 
     private fun setView(queryDetails: QueryDetails) {
-        // TODO: handle multiple groups
-        view.logGroupLabel.text = "Log Group : ${queryDetails.logGroups.first()}"
+        val availableLogGroups = AwsResourceCache.getInstance(project).getResourceNow(
+            CloudWatchResources.LIST_LOG_GROUPS,
+            region = initialQueryDetails.connectionSettings.region,
+            credentialProvider = initialQueryDetails.connectionSettings.credentials
+        ).map { it.logGroupName() }
```

Comment:
Maybe nm the above two comments, since we use connection details here. I'm not entirely sure how all these classes fit together so that might be the issue.